### PR TITLE
add functionality to filter articles by topic

### PR DESCRIPTION
**Pull request by JamesLovesay**

### CODE REVIEW

File: src/components/topic-components/TopicList.jsx [hunk 1]
```diff
@@ -21,6 +21,15 @@ export default function TopicList({ articles, setArticles }) {
         })
     }, [])
 
+    
+    const checkTopic = topics.filter((singleTopic) => {
+        return singleTopic.slug === topic
```
Comment (popatre) [inline src/components/topic-components/TopicList.jsx]: Do you not have an endpoint for this? Instead of a filter?

Comment (popatre) [reply src/components/topic-components/TopicList.jsx]: ?topic="topicHere"

**Review by popatre — CHANGES_REQUESTED** [review]
I've messaged you for a zoom link, to visually see it. 
My only comment is the filter may be unnecessary if you've already got an endpoint you can use. 
Without visually seeing it, can't really offer anymore feedback.

**Review by elijaharley — APPROVED** [review]
Working well now that topics query has been resolved.

**Pull request merged**